src/Compiler/CompilerOpenFPGA_ql.cpp

```diff
@@ -5351,7 +5352,16 @@ bool CompilerOpenFPGA_ql::GenerateIOFloorPlanConstraints() {
   std::string netlistFile = ProjManager()->projectName() + "_post_synth.blif";
   std::string output_path = std::string("--output_path " + ProjManager()->projectName() + "_constraints.xml");
   std::string architectureFile = m_architectureFile.string();
-  std::string command = std::string ("python3 " + 
+  #ifdef _WIN32
```

Comment:
@w0lek : the same logic should hold for WIN32 and Linux, if python is not installed.
option 1: report error
option 2: use bundled python from ipgen (as it is available on Linux as well)

If we know for sure, that the bundled python is good enough for this script (dependencies etc used by the script may change) only then we should use option 2, bundled python.

I would recommend going with option 1 and not use the bundled python (as that is meant to be specific for ipgen + litex support and is a fixed bundle, we cannot add modules into it anytime)

my recommendation would be to make it clear as a requirement that python needs to be installed on Linux/Wndows to use this feature of floor planning contraints.